openlibrary/plugins/openlibrary/js/jquery.toggleText.js

```diff
@@ -0,0 +1,5 @@
+export default function(a, b) {
```

Comment:
Should this be importing jquery? Or are you wanting to do that in a later commit? (There are a few other files which reference but don't import jquery as well).

Reply:
No need. Important to remember that right now jquery is served via the vendor js not webpack and is an implied global.